### ISSUE TITLE
Move to tabbed display

### ISSUE THREAD

**Issue by upstreamplugin** [upstreamplugin/UpStream#341]
Items in the admin area can be hard to find.

It might be worth moving to a tabbed display. Click "Milestones" and they will be directly underneath the top bar. Click "Tasks" and they will be directly there. This way, no scrolling is needed.

It might be wise to move "Activity" to become one of those tabs.

![screen shot 2018-02-05 at 1 37 43 pm](https://user-images.githubusercontent.com/26938537/35822308-e0279ac2-0a79-11e8-8ea3-2dad11ae0ad6.png)

**Comment by upstreamplugin** [comment]
@deenison The more I think about this, the more like the tabbed option for the frontend too.

One big reason ... we should expose the "Activity" area better. Make it easier for people to see what's going on.

Plus, we could have a "Settings" tab where people can control their notifications etc.

**Comment by deenison** [comment]
Agreed 100% in a non-everything-in-one screen and tweaking general UX

**Comment by upstreamplugin** [comment]
I'm adding a Gutenberg tag to this. That might be right time to revisit the UI

**Comment by stevejburge** [comment]
More feedback:

"Not having the ability to keep the 'project details' minimized on start is profoundly annoying because I had to scroll down each time I wanted to get at the only thing that matters in these kinds of apps: the tasks. You should consider letting people re-order these info panels/blocks, or close them outright."

**Comment by stevejburge** [comment]
Building on this ... some images showing this change:

Old:
![Screen Shot 2019-06-04 at 2 25 32 PM](https://user-images.githubusercontent.com/3868028/58903985-a91dc500-86d4-11e9-8956-95780966d9cd.png)

New:
![new](https://user-images.githubusercontent.com/3868028/58903996-acb14c00-86d4-11e9-8946-49b4df148d10.png)

**Comment by sergeidevstar001** [comment]
I checked it in my localhost with recent version, but it works good.
Could you tell me the link to check it?
I think it related with template CSS style.
![image](https://user-images.githubusercontent.com/39214560/58925845-9dd1a480-877b-11e9-867c-ad8e2c7b1d60.png)
If .postbox class have display: inline or inline-block, it will be change like your screenshot.

**Comment by andergmartins** [comment]
@sergeidevstar001 sorry, but it seems like there is a misunderstanding on the task itself. It is asking you to create the tabs, not fixing them (they don't exist right now). Please, read again Steve's comment here https://github.com/upstreamplugin/UpStream/issues/341#issuecomment-498787863 there is a mockup of what you would have to implement. 
Feel free to let us know if you have any question.
